introduction.md

```diff
@@ -0,0 +1,5 @@
+Tristan Libersat
+* Git
+* JS
+* NodeJS
+* PHP
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?